content/04.network_applications.md

```diff
@@ -10,7 +10,31 @@
 
 #### Matrix Factorization
 
-1. Mention techniques for these with some papers
+Matrix factorization is a technique that uses linear algebra to map high dimensional data into a low dimensional space.
+This projection is accomplished by decomposing a matrix into a set of small rectangular matrices.
+In the context of knowledge graphs the input matrix would be an adjacency matrix where the rows and columns are nodes and each entry in this matrix represents the presence of an edge between two nodes.
+The output of this technique would be a small matrix where each row represents a low dimensional embedding for every node in the adjacency matrix.
+Notable methods that perform this projection are Isomap [@doi:10.1126/science.290.5500.2319], Laplacian eigenmaps [@doi:10.7551/mitpress/1120.003.0080] and PCA [@doi:10/bm8dnf] or SVD [@doi:10.1007/BF02288367].
+In this section we discuss selected approaches that utilize the above methods to project knowledge graphs into a low dimensional space.
+
+Laplacian eigenmap is an unsupervised approach that learns the local structure of a knowledge graph.
+Given an adjacency matrix (A) this algorithm first calculates a laplacian matrix.
+A laplacian matrix (L) is a matrix that is calculated by subtracting a knowledge graph's degree matrix (D) (a diagonal matrix where each entry represents the number of edges connected to a node) from its adjacency matrix ($L = D-A$).
+Using the laplacian matrix, this algorithm then calculates eigenvalues and eigenvectors ($Lx = \lambda Dx$).
+The calculated eigenvectors represent the nodes of the knowledge graph [@doi:10.7551/mitpress/1120.003.0080].
+A number of approaches have used variants of this algorithm to perform their own node projection [@raw:gongreplearn; @doi:10.1109/TKDE.2016.2606098; @raw:hanreplearn].
+This type of methodology is easy to execute the only input required is an adjacency matrix and a degree matrix; however, this approach relies on a linear mapping function, which may not be appropriate given certain biomedical problems.
+
+Following Laplacian eigenmaps, other works constructed variants of the adjacency matrix before undergoing matrix factorization [@doi:10.1145/2806416.2806512; @raw:zhaoreplearn; @raw:yangreplearn; @raw:tureplearn].
```

Comment:
My understanding from this description is that all of these methods use random walk and then do matrix factorization on the matrix after some iterative random walk procedure. Is something like that correct?